clouddriver-ecs/src/test/groovy/com/netflix/spinnaker/clouddriver/ecs/services/ContainerInformationServiceSpec.groovy

```diff
@@ -173,15 +232,19 @@ class ContainerInformationServiceSpec extends Specification {
     retrievedHealthStatus == expectedHealthStatus
 
     where:
-    lastStatus       | resultStatus
-    'PROVISIONING'   | 'Starting'
-    'PENDING'        | 'Starting'
-    'ACTIVATING'     | 'Starting'
-    'RUNNING'        | 'Unknown'
-    'DEACTIVATING'   | 'Down'
-    'STOPPING'       | 'Down'
-    'DEPROVISIONING' | 'Down'
-    'STOPPED'        | 'Down'
+    healthStatus  | resultStatus  | lastStatus
+    'UNKNOWN'     | 'Starting'    | 'PROVISIONING'
+    'UNKNOWN'     | 'Starting'    | 'PENDING'
+    'UNKNOWN'     | 'Starting'    | 'ACTIVATING'
+    'UNKNOWN'     | 'Starting'    | 'RUNNING'
+    'UNHEALTHY'   | 'Down'        | 'PROVISIONING'
+    'UNHEALTHY'   | 'Down'        | 'PENDING'
+    'UNHEALTHY'   | 'Down'        | 'ACTIVATING'
+    'UNHEALTHY'   | 'Down'        | 'RUNNING'
+    'HEALTHY'     | 'Starting'    | 'PROVISIONING'
+    'HEALTHY'     | 'Starting'    | 'PENDING'
+    'HEALTHY'     | 'Starting'    | 'ACTIVATING'
+    'HEALTHY'     | 'Unknown'     | 'RUNNING'
```

Comment:
I might be confused, but why is the resulting status of HEALTHY + RUNNING = 'Unknown' ?

Reply:
I didn't want to change the current logic for a running task in general (not sure what's relying on it and it was previously 'Unknown'

Reply:
(adding for context), see comment above re: the var being set my a method that doesn't actually return 'up'